karma.conf.js

```diff
@@ -1,13 +1,11 @@
 module.exports = function(config) {
-  const jQueryVersion = process.env.jqueryOld === 'true' ? 'js/tests/vendor/jquery-1.9.1.min.js' : 'assets/js/vendor/jquery-slim.min.js'
+  const jqueryFile = process.env.jqueryOld === 'true' ? 'js/tests/vendor/jquery-1.9.1.min.js' : 'assets/js/vendor/jquery-slim.min.js'
   config.set({
     frameworks: ['qunit'],
     plugins: ['karma-chrome-launcher', 'karma-qunit'],
-
-
     // list of files / patterns to load in the browser
     files: [
-      jQueryVersion,
+      jqueryFile,
       'assets/js/vendor/popper.min.js',
       'js/dist/util.js',
```

Comment:
Would it work if we did `js/dist*.js` and just excluded `index.js`? Or even just include everything?

Reply:
Nope because we have to be sure `util.js` is the first included script